CHANGES.md

```diff
@@ -1,5 +1,67 @@
 # Changelog
 
+# 1.5.10 / 0.21.10 (libraries)
+
+### New
+
+- [asset backfills] Previously, when partitions definitions were changed after backfill launch, the asset backfill page would be blank. Now, when partitions definitions are changed, the backfill page will display statuses by asset.
+- Added a new `MetadataValue.job` metadata type, which can be used to link to a Dagster job from other objects in the UI.
```

Comment:
nit: i think its clearer to order the non `[]` prefixed entries above the others

Reply:
https://github.com/dagster-io/dagster/pull/18412/files#r1411319165